src/sentry/models/grouphashmetadata.py

```diff
@@ -2,15 +2,17 @@
 from django.utils import timezone
 
 from sentry.backup.scopes import RelocationScope
-from sentry.db.models import FlexibleForeignKey, Model, region_silo_model
+from sentry.db.models import Model, region_silo_model
 
 
 @region_silo_model
 class GroupHashMetadata(Model):
     __relocation_scope__ = RelocationScope.Excluded
 
     # GENERAL
-    grouphash = FlexibleForeignKey("sentry.GroupHash", unique=True, related_name="metadata")
+    grouphash = models.OneToOneField(
+        "sentry.GroupHash", unique=True, related_name="_metadata", on_delete=models.CASCADE
```

Comment:
is `unique=True` still necessary on a OneToOne field?

Reply:
I think it's implicit and we can remove it

Reply:
Yup, you're both right - turns out it's not necessary, because it's [set by default](https://github.com/django/django/blob/a57596e443ecb67140e1a9fc0f0e87446b2d0174/django/db/models/fields/related.py#L1235). Good catch! I'll take it out.